examples/gobot_tutorial.ipynb

```diff
@@ -1542,159 +661,104 @@
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "-Xf1gCdmgSBr"
+   },
+   "outputs": [],
    "source": [
-    "bot(['chinese food'])"
+    "bot([\"nice idea, thanks!\"])"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "['Sure, the lucky star is on cambridge leisure park clifton way cherry hinton.']"
-      ]
-     },
-     "execution_count": 14,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Ad_GDanAgSCi"
+   },
+   "outputs": [],
    "source": [
-    "bot(['thanks, give me their address'])"
+    "bot.reset()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "['The phone number of the lucky star is 01223 244277.']"
-      ]
-     },
-     "execution_count": 15,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "r5312Df1gSC0"
+   },
+   "outputs": [],
    "source": [
-    "bot(['i want their phone number too'])"
+    "bot(['hi bot'])"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "['You are welcome!']"
-      ]
-     },
-     "execution_count": 16,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "jXQw_11jgSDM"
+   },
+   "outputs": [],
    "source": [
-    "bot(['bye'])"
+    "bot([\"looks ok, the sun is bright and yesterday's rain stopped already\"])"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "-SKJicTCgSDe"
+   },
    "outputs": [],
    "source": [
-    "bot.reset()"
+    "bot(['i dont wanna'])"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 18,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "['What kind of food would you like?']"
-      ]
-     },
-     "execution_count": 18,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "cell_type": "markdown",
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "wdg4wl0dgSD9"
+   },
    "source": [
-    "bot(['hi, is there any cheap restaurant?'])"
+    "You can also train a more advanced goal-oriented bot following [gobot_extended_tutorial.ipynb](https://github.com/deepmipt/DeepPavlov/blob/master/examples/gobot_extended_tutorial.ipynb)"
```

Comment:
Will it work like this?
```suggestion
    "You can also train a more advanced goal-oriented bot following [gobot_extended_tutorial.ipynb](gobot_extended_tutorial.ipynb)"
```

Reply:
In COLAB relative links seem to not work.